Cargo.toml

```diff
@@ -17,21 +17,20 @@ resolver = "2"
 
 [patch.crates-io]
 # We should usually track the `p2pderivatives/feature/ln-dlc-channels[-10101]` branch
-dlc-manager = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "9815582" }
-dlc-messages = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "9815582" }
-dlc = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "9815582" }
-dlc-sled-storage-provider = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "9815582" }
-p2pd-oracle-client = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "9815582" }
-dlc-trie = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "9815582" }
-simple-wallet = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "9815582" }
+dlc-manager = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "d1e08a0" }
```

Comment:
Note: we will need to bump this once more because this fix landed: https://github.com/p2pderivatives/rust-dlc/pull/162